user_sync/rules.py

```diff
@@ -780,14 +812,14 @@ def read_remove_list(file_path, delimiter = None, logger = None):
             id_type = row.get(id_type_column_name)
             user = row.get(user_column_name)
             domain = row.get(domain_column_name)
-            user_key = RuleProcessor.get_user_key(user, domain, id_type)
+            user_key = RuleProcessor.get_user_key(user, domain, None, id_type)
```

Comment:
Right, I get it.  The bug is that get_user_key doesn't specify domain and email as optional.  You should actually fix that (on line 721) and then call get_user_key with keyword args here.  But just leave it for now and I'll fix it everywhere as a separate change.

Be sure to put in a comment saying "specifying None for (optional) email, because we don't have it".